tla/tlc.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+#
+# Python TLC wrapper script for the CCF project
+# Goals:
+# - No dependencies, no venv, no pip install
+# - Set sensible defaults with an eye on performance
+# - Capture useful switches for CI, debugging
+# - Expose specification configuration through CLI
+# - Provide a useful --help, and basic sanity checks
+
+import os
+import argparse
+import pprint
+import pathlib
+
+DEFAULT_JVM_ARGS = [
+    "-XX:+UseParallelGC",  # Use parallel garbage collection, for performance
+]
+
+
+DEFAULT_CLASSPATH_ARGS = ["-cp", "tla2tools.jar:CommunityModules-deps.jar"]
+
+
+def cli():
+    parser = argparse.ArgumentParser(
+        description="TLC model checker wrapper for the CCF project"
+    )
+
+    # Common options for all commands
+    parser.add_argument(
+        "-v",
```

Comment:
possible to use another letter? maybe its just me but I associate `-v` with either get version or verbose mode.

Reply:
also is it worth enabling this by default? could be useful for ci runs

Reply:
It's exactly verbose, in my mind. And the reason it's not enabled is because it's verbose, and the env is (very) large, and the TLC invocations are already extremely verbose.
Maybe a middle of the road option would be to print the formatted command an env diff, which shouldn't be too bad.

Reply:
set calls this `-x`. Agreed, I didn't mean all of env, just the ones we are interested in

Reply:
-x it is